Sources/PointFreeRouter/PointFreeRouter.swift

```diff
@@ -0,0 +1,35 @@
+import ApplicativeRouter
+import Foundation
+
+public struct PointFreeRouter {
+  public let baseUrl: URL
+  public let router: Router<Route>
+
+  public init(baseUrl: URL = URL(string: "http://localhost:8080")!) {
+    self.baseUrl = baseUrl
+    self.router = routers.reduce(.empty, <|>)
+  }
+
+  public init(baseUrl: URL = URL(string: "http://localhost:8080")!, router: Router<Route>) {
```

Comment:
I can't do a default argument of `router: Router<Route> = routers.reduce(.empty, <|>)` since `routes` is internal :/